tests/test1.conf

```diff
@@ -26,4 +26,8 @@ tables = {
    `the_blobs` = {
```

Comment:
Hmm weird, the `the_blobs` configuration/table seems to be breaking this fix, it passes if I remove this part:

```diff
diff --git a/tests/test1.conf b/tests/test1.conf
index 5827ba6..07f1bc3 100644
--- a/tests/test1.conf
+++ b/tests/test1.conf
@@ -23,9 +23,6 @@ tables = {
      `c`    = fixed ''
      `name` = texthash 10
    }
-   `the_blobs` = {
-     `blob1` = fixed unquoted '0x676f6f64627965'
-   }
    `test_with_column_names` = {
      `a` = fixed '111'
      `b` = fixed '222'
diff --git a/tests/test1_anon.sql b/tests/test1_anon.sql
index ee15549..1c1c47b 100644
--- a/tests/test1_anon.sql
+++ b/tests/test1_anon.sql
@@ -143,7 +143,7 @@ CREATE TABLE `the_blobs` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
 
 INSERT INTO `the_blobs` VALUES (
-  0x676f6f64627965,
+  0x68656c6c6f,
   0x68656c6c6f,
   0x68656c6c6f,
   0x68656c6c6f
```

Not entirely sure why at the moment :thinking:

Reply:
I will have a look later today, stay tuned;-)

Reply:
I appreciate it, thank you!

Reply:
I was able to run your patch successfully, using the following test1.zip (test1.sql/test1_anon.sql files). The test1.sql is a little bit closer to the output of a real mysqldump command. 

I have plan to review flex state transitions to have better support for those manual editions (but I think this is out of scope of this PR).

In the meanwhile, I think you can just update the PR with theses two files, and the 'make check' will be ok ;-)

[test1.zip](https://github.com/ppomes/myanon/files/9143460/test1.zip)

Reply:
I just updated the tests files, CI is now ok!

Reply:
Thanks - sorry I didn't get to it yet! :+1: